src/C-DEngine/C-DEngines/C-DNMIService/cde_NMIAPI.cs

```diff
@@ -2408,7 +2408,7 @@ public static TheFormInfo CreateEngineForms(ICDEThing pBaseThing, Guid pID, stri
         /// <param name="pDeviceTypeDefault">List of device types for the table of things</param>
         /// <param name="pDeviceTypeOptions">Default Device Type for new entries in the table of things</param>
         /// <returns></returns>
-        public static cdeConcurrentDictionary<string,TheMetaDataBase> CreateEngineForms(TheThing pBaseThing, Guid pID, string pTitle, string pFilter, int pFldOrder, int pFlag, int pACL, string category, string pCustomCommand, bool AddAddress, string pDeviceTypeOptions = null, string pDeviceTypeDefault = null)
+        public static cdeConcurrentDictionary<string,TheMetaDataBase> CreateEngineForms(TheThing pBaseThing, Guid pID, string pTitle, string pFilter, int pFldOrder, int pFlag, int pACL, string category, string pCustomCommand, bool AddAddress, string pDeviceTypeOptions = null, string pDeviceTypeDefault = null, bool AddNodeName = false)
```

Comment:
The function is public and even though its preset existing plugins will fail as the original function signature is not there.

Reply:
Yep, fixed.